src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java

```diff
@@ -49,6 +53,8 @@
 import javafx.scene.control.MenuItem;
 import javafx.scene.control.MultipleSelectionModel;
 import javafx.scene.control.SelectionMode;
+import javafx.scene.control.Spinner;
+import javafx.scene.control.SpinnerValueFactory.IntegerSpinnerValueFactory;
```

Comment:
imports can be deleted. (with eclipse "organize imports")

Reply:
Deleted unused imports